js/index.js

```diff
@@ -40,3 +40,101 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// nav
+// class selector
+const navClass = document.querySelectorAll('nav')
+// anchor selector
+const navText = document.querySelectorAll('a');
+
+// change all nav text to green
+for (let i = 0; i < navText.length; i++) {
+  navText[i].style.color = "green";
+  // item content iterated
+  navText[i].textContent = siteContent['nav'][`nav-item-${i + 1}`]
+}
+
+// cta
+// class selector
+const ctaClass = document.querySelectorAll('.cta')
+
+  // h1
+  const ctaH1 = document.querySelector('h1');
+  ctaH1.textContent = siteContent['cta']['h1'];
+
+  // button
+  const ctaButton = document.querySelector('button');
+  ctaButton.textContent = siteContent['cta']['button'];
+
+  // img
+  const ctaImg = document.querySelector("#cta-img");
+  ctaImg.src= siteContent["cta"]["img-src"];
+
+// main-content 
+// class selector
+const mainClass = document.querySelectorAll('.main-content')
+// heading selector
+const mainHeading = document.querySelectorAll('.main-content h4');
+// content selector
+const mainParagraphs= document.querySelectorAll('.main-content p');
+// image selector
+const middleImg = document.querySelector('#middle-img');
+
+for(let i = 0; i < mainParagraphs.length; i++){
+  let section = '';
+  switch(i){
+    case 0: section = 'features'
+    case 1: section = 'about'
+    case 2: section = 'services'
+    case 3: section = 'product'
+    case 4: section = 'vision'
+  }
+  mainHeading[i].textContent = siteContent["main-content"][`${section}-h4`]
+  mainParagraphs[i].textContent = siteContent["main-content"][`${section}-content`]
+
+  // img
+  middleImg.src = siteContent["main-content"]["middle-img-src"];
+}
+
+// contact
+// class selector
+const contactClass = document.querySelectorAll('.contact')
+
+// heading selector
+const contactH1 = document.querySelector('.contact h4');
+// content selector
+const contactParagraphs = document.querySelectorAll('.contact p')
+
+  // heading
+  contactH1.textContent = siteContent['contact']['contact-h4'];
+
+  // content
+  contactParagraphs[0].textContent = siteContent['contact']['address']
+  contactParagraphs[1].textContent = siteContent['contact']['phone']
+  contactParagraphs[2].textContent = siteContent['contact']['email']
+
+// footer
+// content selector
+const footerParagraphs = document.querySelector('footer p')
+
+// content
+footerParagraphs.textContent = siteContent['footer']['copyright']
+
+// new nav items
+const navBar = document.querySelector ('nav');
+
+const firstAnchor = document.createElement('a');
+firstAnchor.setAttribute("href", "#");
+firstAnchor.textContent = 'to the front..';
+firstAnchor.style.color = 'green';
+console.log(firstAnchor)
```

Comment:
Good job console logging when you can. When you deploy projects into production make sure you delete these console logs. It's a good habit to pick up early.